cxx-squid/src/main/java/org/sonar/cxx/CxxVCppBuildLogParser.java

```diff
@@ -83,15 +83,15 @@ public void parseVCppLog(File buildLog, String baseDir, String charsetName) {
               }
             }
 
-          if(line.contains("\\V100\\Microsoft.CppBuild.targets")) {
-            platformToolset = VSVersion.V100;
-          } else if(line.contains("\\V110\\Microsoft.CppBuild.targets")) {
-            platformToolset = VSVersion.V110;
-          } else if(line.contains("\\V120\\Microsoft.CppBuild.targets")) {
-            platformToolset = VSVersion.V120;
-          } else if(line.contains("\\V140\\Microsoft.CppBuild.targets")) {
-            platformToolset = VSVersion.V140;
-          }
+        if (line.contains("\\V100\\Microsoft.CppBuild.targets") || line.contains("Microsoft Visual Studio 10.0\\VC\\bin\\CL.exe")) {

```

Comment:
I will remove it if you are fine with it. This will be a restriction for the feature and it will only work with the default Visual Studio installation paths.